site/src/components/DAUChart/DAUChart.test.tsx

```diff
@@ -1,37 +0,0 @@
-import { render } from "testHelpers/renderHelpers"
```

Comment:
Is this deletion intended?

Reply:
Yeap, it is just a render test which should be on the storybook.